custom-instrumentation/segments/functions.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const { someCbTask } = require('./functions/callback.js')
```

Comment:
this is fine but you can put this in an index.js within `functions` folder and you can still require it as `require('./functions')`